docs/20260104_budget-scale-visualization.md

```
@@ -0,0 +1,174 @@
+| 10 | 1,024倍 | 153m | 1:500K | 市区町村 |
+| 13 | 8,192倍 | 19m | 1:70K | 村・郊外 |
+| 15 | 32,768倍 | 4.8m | 1:15K | 道路 |
+| 17 | 131,072倍 | 1.2m | 1:4K | 建物・住所 |
+| 19 | 524,288倍 | 0.3m | 1:1K | 交差点詳細 |
+| 20 | 1,048,576倍 | 0.15m | 1:500 | 中規模建物 |
+
+**注**: m/pixel は赤道上の値。緯度により cos(緯度) を乗算する必要がある。
+
+### スケール計算式
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)